build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -131,10 +143,12 @@ private static bool CheckIfAllBuildsHavePublished()
                  { "Windows_x86", false },
                  { "Windows_x64", false },
                  { "Ubuntu_x64", false },
```

Comment:
Yes. But not as part of this work. This would be yet a lot more work and we need to get this in. Plus we need to understand what changing the version now would mean to old install scripts and things like that. I don't want to make that a requirement for this PR as we need to get this in.